src/js/trove/particle.js

```diff
@@ -70,12 +67,38 @@ define(["js/runtime-util", "js/ffi-helpers", "trove/json", "trove/world", "trove
 
 
           //////////////////////////////////////////////////////////////////////
-          
-          var ToParticle = function(handler, ename, sdict) {
+
+          var sendEvent = function(ename, data, options) {
+            var xhr = new XMLHttpRequest();
```

Comment:
It hasn't been a problem for me, but then again, I haven't been doing anything ultra-fancy with it.